lib/improver/wind_downscaling.py

```diff
@@ -711,13 +712,13 @@ def find_coord_names(self, cube):
         try:
             xname = cube.coord(axis="x").name()
         except CoordinateNotFoundError as exc:
-            print("'{0}' while xname setting. Args: {1}.".format(exc.message,
-                                                                 exc.args))
+            print(("'{0}' while xname setting. Args: {1}.".format(exc,
```

Comment:
Do we need these double brackets? It seems like this already had new style print statements

Reply:
Done.